scripts/tests/app/web/ionic-authentication.json

```diff
@@ -4,6 +4,5 @@
   "ui": "ionic",
   "layout": "side-menu",
   "auth": true,
-  "lazy": false,
-  "location": "hash"
```

Comment:
keep hash option tested as it's needed in some cases

Reply:
reverted :) hash works on ionic now too